tests/integration/test_charm.py

```diff
@@ -8,29 +8,37 @@
 import logging
 
 import pytest
-from pytest import Config
+import pytest_asyncio
+import requests
 from pytest_operator.plugin import OpsTest
 
 logger = logging.getLogger(__name__)
 
+APP_NAME = "flask-k8s"
```

Comment:
I'm wondering if `APP_NAME` is required, would it be better if we deploy the application `application = model.deploy(...)` and then use the name from the application directly: `application.name`.

Reply:
Good idea, updated, thanks!